app/models/sipity/workflow_role.rb

```diff
@@ -9,5 +9,12 @@ class WorkflowRole < ActiveRecord::Base
     has_many :workflow_responsibilities, dependent: :destroy, class_name: 'Sipity::WorkflowResponsibility'
     has_many :workflow_state_action_permissions, dependent: :destroy, class_name: 'Sipity::WorkflowStateActionPermission'
     has_many :entity_specific_responsibilities, dependent: :destroy, class_name: 'Sipity::EntitySpecificResponsibility'
+
+    # @todo This is a hack; I don't want to include reference to the admin set; However based on the current UI this is required.
```

Comment:
Can you update this comment and explain why "based on the current UI this is required"?